src/geotop/pedo.funct.cc

```diff
@@ -140,12 +138,13 @@ double k_hydr_soil(double psi, double ksat, double imp, double i, double s,
 
   double k,TETA,psisat;
 
-  psisat = (pow((pow(1.0-i/(s-r),-1.0/m)-1.0),1.0/n))*(-1.0/a);
-  TETA = 1.0/pow((1.0+pow(a*(-std::min<double>(psisat,psi)),n)),m);
+  psisat = (power((power(1.0-i/(s-r),-1.0/m)-1.0),1.0/n))*(-1.0/a); // power() works but NOT Padè approximation
+  TETA = 1.0/pow((1.0+pow(a*(-std::min<double>(psisat,psi)),n)),m); // power() does NOT work
 
-  k = ksat * pow(TETA,v)*(pow((1-pow((1-pow(TETA,(1.0/m))),m)),2.0));
+  k = ksat * pow(TETA,v) * pow((1-pow((1-pow(TETA,(1.0/m))),m)),2.0); // power() does NOT work
```

Comment:
when you see `pow(...., 2.0)` please use something smart.

if you will you may also write a macro `pow_2` like this
```
#define pow_2(a) a*a
```

Reply:
I haven't changed the pow(N,2) yet!
I like this approach; but in this case this new function would be always active, even without the flags MATH_OPTIM, right?

Reply:
Sobstituing this new pow in **meteo.cc** the energy balance does NOT converge in some 1D cases (i.e, Matsch_B2_Ref_007). I don't know why. How does this macro handle different type of variables? Should it work for all types, right?

Anyway If i write
```
template <class T>
T pow_2(const T a){
#ifdef MATH_OPTIM
return a*a;
#else
return std::pow(a,2);
#endif
}
```
it works....

Reply:
Now I added the parenthesis ``` #define pow_2(a) (a*a)``` and it works! 
Is it "normal"? :)